snc.sh

```diff
@@ -222,8 +222,6 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo mkdir /opt/release-manife
 retry ${OC} -n openshift-cluster-version get pods -o=name
 CVO_POD_NAME=$(${OC} -n openshift-cluster-version get pods -o=name)
 retry ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "sudo KUBECONFIG=/opt/kubeconfig oc rsync -n openshift-cluster-version ${CVO_POD_NAME}:/release-manifests/ /opt/release-manifests/"
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo sed -i "s/replicas: 2/replicas: 1/" /opt/release-manifests/*deployment.yaml'
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo sed -i "s/replicas: 2/replicas: 1/" /opt/release-manifests/*clusterserviceversion.yaml'
```

Comment:
I still see this in the 4.8.2 release image. :thinking: 

```
[root@6cbaf2e1b239 release-manifests]# grep -R "replicas: 2" .
./0000_50_olm_15-packageserver.clusterserviceversion.yaml:            replicas: 2
```

Reply:
@guillaumerose can you check 4.9 night side because there it is failing https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/5093/pull-ci-openshift-installer-master-e2e-crc/1419751100604485632

```
Warning: Permanently added 'api.crc.testing,192.168.126.11' (ECDSA) to the list of known hosts.
+ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_ecdsa_crc core@api.crc.testing 'sudo sed -i "s/replicas: 2/replicas: 1/" /opt/release-manifests/*clusterserviceversion.yaml'
Warning: Permanently added 'api.crc.testing,192.168.126.11' (ECDSA) to the list of known hosts.
sed: can't read /opt/release-manifests/*clusterserviceversion.yaml: No such file or directory
```

Reply:
OK I understand. In 4.9, they introduced a package server manager which takes care of this.

https://github.com/openshift/operator-framework-olm/blob/1dc76f08ed05a635458420ffa979aebbe59a3890/pkg/package-server-manager/config.go#L14

So looks good for 4.9 :+1: